.travis.yml

```diff
@@ -55,13 +56,19 @@ jobs:
         - bash -x -e ./scripts/travis/06-run-config-tests.sh
 
     - stage: test
-      name: Run tests with composer build
+      name: Run tests with composer (max) build
 
     - stage: test
       name: Run tests with drush_make build
       install:
         - bash -x -e ./scripts/travis/03-build-thunder-drush.sh
 
+    - stage: test
+      name: Run tests with composer (min) build
+      env: PREFER_LOWEST="--prefer-lowest"
```

Comment:
It seems strange, that the value of the variable is basically either the name of it or empty.
I would expect PREFER_LOWEST to be a boolean. Alternatively, we could name it more generally ADDITIONAL_COMPOSER_PARAMETERS, or something along these lines. 
But PREFER_LOWEST=--prefer-lowest is strange